### PR TITLE
Portfolio: setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by Brenda309**
### In this pull request, I:

- [x] created the toolbar
- [x] created the headline section

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 24px;
+  color: #344563;
+  margin: 12px 0;
+}
```
Comment (keroloussamy) [inline style.css:41]: - [ ] Kindly change the font for this paragraph to make it like the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: style.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #7f8cff;
+  margin: 0 8px;
+}
```
Comment (keroloussamy) [inline style.css:51]: - [ ] Kindly change the font for this `h2` to make it like the Figma design and the color and remove the underline.

| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: README.md [hunk 3]
```diff
@@ -2,55 +2,42 @@
...
 ## Authors
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [Brenda309](https://github.com/githubhandle)
+- Twitter: [@Brenda](https://twitter.com/twitterhandle)
+- LinkedIn: [Wihogora Brenda](https://linkedin.com/in/linkedinhandle)
```
Comment (keroloussamy) [inline README.md:39]: - [ ] Kindly change the link to your actual link, so anyone can contact you.

Like
- GitHub: `[Brenda309](https://github.com/Brenda309)`

File: README.md [hunk 4]
```diff
@@ -2,55 +2,42 @@
 
 # Project Name
```
Comment (keroloussamy) [inline README.md:4]: - [ ] Kindly change this title to the project name.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @Brenda309, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️

Hi @Brenda309, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there

 ### Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [x] In future let's make sure we use descriptive commit messages - each commit should be small enough to describe it in one sentence. The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8).
Your commit messages are repetitive, are not capitalized and are not descriptive. You cannot change them now, but please consider writing more meaningful commit messages from now on.
- [x] The name of your portfolio is at the moment "mobile-portfolio", but next week we will make it responsive and it will adapt to desktop view as well. Because of this, you can consider renaming your project something more meaningful. As simple as "Portfolio" or "Personal Portfolio" will do. If you rename it, don't forget to edit your README as too.
- [ ] Please consider adding `.gitignore` file to your repo and in that file, add `node_modules/`.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**